### PR TITLE
include info about custom contract interactions

### PR DESCRIPTION
**Pull request by critesjosh**
### Description

These additions address the lack of information around using contractkit for interacting with custom smart contracts.

also some formatting fixes

### CODE REVIEW

File: packages/docs/developer-resources/walkthroughs/hello-contract-remote-node.md [hunk 1]
```diff
@@ -74,14 +88,13 @@ module.exports = function(deployer) {
...
+When you deploy contracts to the Celo network with a remote node, you have to sign the contract deployment transaction locally before sending it to the remote node to be broadcast to the network. This presents some unique challenges when using Ethereum development tools \(like Truffle\) because Celo transaction objects are slightly different than Ethereum transaction objects.
 
-When you are ready to deploy your contract to Alfajores, you'll need a Celo client connected to the testnet. In this exercise we are going to connect to a remote node to read and write to the public test net, but you could also run a test net node locally to perform the same actions.
+When you are ready to deploy your contract to Alfajores, you'll need a Celo client connected to the testnet. In this exercise we are going to connect to a remote node to read and write to the public testnet (Alfajores), but you could also run a testnet node locally to perform the same actions.
```
Comment (eelanagaraj) [inline packages/docs/developer-resources/walkthroughs/hello-contract-remote-node.md]: nit: could change "we are going to" -> "you will" or "This exercise illustrates how to connect ..."

File: packages/docs/developer-resources/walkthroughs/hello-contract-remote-node.md [hunk 2]
```diff
@@ -157,21 +177,21 @@ Then, in the `networks` object, we can add the initialized `kit`provider to an `
...
 
 ```javascript
-$ truffle migrate --network alfajores
+truffle migrate --network alfajores
 ```
 
 #### Custom Node.js Deployment
 
-In this section, we will go over how to deploy a contract using a simple Node.js script to show how you can do it without using Truffle.
+In this section, you deploy a contract using a simple Node.js script to show how you can do it without using Truffle.
```
Comment (eelanagaraj) [inline packages/docs/developer-resources/walkthroughs/hello-contract-remote-node.md]: nit: "you will deploy" instead of "you deploy"?

**Review by eelanagaraj — APPROVED** [review]
Looks good! Left two nit comments but up to you if you want to add those changes or not

**Comment by critesjosh** [comment]
thanks @eelanagaraj! i added the changes

**Pull request merged**